### ISSUE TITLE
/web/notifications is confusing

### ISSUE THREAD

**Issue by matheod** [tootsuite/mastodon#2624]
_TL;DR : /web/notifications url is confusing, show two notification column with no return link at the top of the third column._

I clicked on a tab and was surprised by the fact there was two column "notification" (the third column was replaced by notification column), and there was no "return" link at the top of the third column to go back to global/local timeline. I though of a bug so I reloaded the page and it was still here. it took me a while to notice that I was on the url /web/notifications. I have no idea how I ended here, but this is very confusing.

It don't see the point of this url, show twice the notification doesn't seem usefull, and having no "return" link is very confusing.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by mjankowski** [comment]
I don't really understand what you are describing here.  Can you elaborate more on what the problem is?

If you are in a layout that's large enough to have multiple columns at once, you shouldn't see the notifications navigation item. If you are in a layout small enough to see it, the notifications column should just always be there, and you shouldn't see the nav option for it.

I can sort of replicate what you are describing by going directly to that URL in a large browser window, or resizing my browser when I'm on that URL ... but that doesn't seem like a problem?

**Comment by Gargron** [comment]
You can open any column using the URL. This is how the single-column layout works primarily, and how the "context column" works on desktop.
